bin/node-pg-migrate

```diff
@@ -306,27 +311,21 @@ function readJson(json) {
   }
 }
 
-try {
-  // Load config (and suppress the no-config-warning)
-  const oldSuppressWarning = process.env.SUPPRESS_NO_CONFIG_WARNING
-  process.env.SUPPRESS_NO_CONFIG_WARNING = 1
-  const config = require('config') // eslint-disable-line global-require,import/no-extraneous-dependencies
-  if (config.has(argv[configValueArg])) {
-    const db = config.get(argv[configValueArg])
-    readJson(db)
-  }
-  process.env.SUPPRESS_NO_CONFIG_WARNING = oldSuppressWarning
-} catch (err) {
-  if (err.code !== 'MODULE_NOT_FOUND') {
-    throw err
-  }
+// Load config (and suppress the no-config-warning)
+const oldSuppressWarning = process.env.SUPPRESS_NO_CONFIG_WARNING
+process.env.SUPPRESS_NO_CONFIG_WARNING = 1
+const config = tryRequire('config')
+if (config && config.has(argv[configValueArg])) {
```

Comment:
**nitpick (non-blocking):** If we have newest JS support here, we can make benefit of optional chaining

```suggestion
if (config?.has(argv[configValueArg])) {
```

Reply:
The `node-pg-migrate` "binary" does not currently transpille and we officially support node v10, therefore unless we turn this file into TS, we can hardly do anything about it.